builders/README.md

```diff
@@ -14,6 +14,10 @@ any machine that can run Docker can do the cross-compilation.
 
 ## Changelog
 
+**Version 0017:**
```

Comment:
Okay, now we need to bump the version in `builder/Makefile` too:

```
BUILDERS_PREFIX := cosmwasm/go-ext-builder:0016
```

This then ensures a rebuild of the builder images is tagged correctly.